projects/core/README.md

```diff
@@ -0,0 +1,75 @@
+# Clarity Core Web Components [![npm version](https://badge.fury.io/js/%40clr%2Fcore.svg)](https://badge.fury.io/js/%40clr%2Fcore)
+
+Clarity Core is a suite of Web Components for [Clarity Design System](https://clarity.design/storybook/core).
+
+## Quick Start Install
+
+1.  First, install the Clarity Core package from npm.
```

Comment:
The info in this file is irrelevant now. It should either be updated, or the file deleted.